### PR TITLE
1422 and 1423 Remove HDFS and Oozie from Menas

### LINKED ISSUE #1422: Remove hard dependency on HDFS from Menas
## Background
We want to go into the cloud where this might not be possible, or we might choose different storage.

## Feature
Remove hard dependency on HDFS from Menas
- it is not needed to run menas
- HDFS browsers in menas are disabled

### LINKED ISSUE #1423: Remove Oozie from Menas
## Background
We will be using a different scheduler then Oozie in the cloud.

## Feature
Remove Oozie from Menas

### PR DESCRIPTION
**Pull request by Zejnilovic**
Removing HDFS and Oozie from Menas, dao and data-model

All this is for the containerization and moving Menas to cloud

Closes #1422 
Closes #1423

### CODE REVIEW

File: data-model/src/main/scala/za/co/absa/enceladus/model/menas/HDFSFolder.scala [hunk 1]
```diff
@@ -15,9 +15,9 @@
 
 package za.co.absa.enceladus.model.menas
 
-case class HDFSFolder
+case class HDFSFolder2
```
Comment (AdrianOlosutean) [inline data-model/src/main/scala/za/co/absa/enceladus/model/menas/HDFSFolder.scala:18]: Why HDFSFolder2? Shouldn't it be deleted?

Comment (Zejnilovic) [reply data-model/src/main/scala/za/co/absa/enceladus/model/menas/HDFSFolder.scala]: Good catch

File: menas/ui/css/style.css [hunk 2]
```diff
@@ -82,17 +82,3 @@ html, body {
...
-  margin-top: -1rem !important;
-  position: absolute !important;
-}
-
-.scheduleMenuHBox {
-  height: 2rem !important;
-  margin-top: 1rem !important;
-}
-
-h5[id$=datasetDetailView--scheduleTimingTitle] {
```
Comment (benedeki) [inline menas/ui/css/style.css:96]: `h5` can go too?

File: data-model/src/main/scala/za/co/absa/enceladus/model/menas/HDFSFolder.scala (hunk 1 above)
[the remark this answers is not in the document]
Comment (Zejnilovic) [reply data-model/src/main/scala/za/co/absa/enceladus/model/menas/HDFSFolder.scala]: This was already removed

File: data-model/src/main/scala/za/co/absa/enceladus/model/Dataset.scala [hunk 3]
```diff
@@ -44,8 +43,7 @@ case class Dataset(
   dateDisabled: Option[ZonedDateTime] = None,
   userDisabled: Option[String]        = None,
   conformance:  List[ConformanceRule],
-  parent:       Option[MenasReference] = None,
-  schedule:     Option[OozieSchedule] = None) extends VersionedModel with Auditable[Dataset] {
```
[the remark this answers is not in the document]
Comment (Zejnilovic) [reply data-model/src/main/scala/za/co/absa/enceladus/model/Dataset.scala:48]: should be ignored

File: menas/ui/css/style.css (hunk 2 above)
Comment (Zejnilovic) [reply menas/ui/css/style.css:96]: It is H5 for a scheduler

**Comment by Zejnilovic** [comment]
> What about changing the project version in the main pom to make the build recognizable from regular branch?

Smart. Added AWS qualifier. Adheres to semver.

**Pull request merged**